price-feeder/oracle/oracle_test.go

```diff
@@ -697,3 +697,94 @@ func TestGetComputedPricesTickersConversion(t *testing.T) {
 		prices[btcPair.Base],
 	)
 }
+
+func TestGetComputedPricesEmptyTvwap(t *testing.T) {
+	symbolUSDT := "USDT"
+	symbolUSD := "USD"
+	symbolDAI := "DAI"
+	symbolETH := "ETH"
+
+	pairETHtoUSDT := types.CurrencyPair{
+		Base:  symbolETH,
+		Quote: symbolUSDT,
+	}
+	pairETHtoDAI := types.CurrencyPair{
+		Base:  symbolETH,
+		Quote: symbolDAI,
+	}
+	pairETHtoUSD := types.CurrencyPair{
+		Base:  symbolETH,
+		Quote: symbolUSD,
+	}
+	basePairsETH := []types.CurrencyPair{
+		pairETHtoUSDT,
```

Comment:
Can we make this into a table-based test?

Reply:
done at https://github.com/umee-network/umee/pull/1262/commits/33ed32a79d32d47b5de3bbd9b06ec7e3a6a5c298